extensions/parquet-extensions/src/main/java/io/druid/data/input/parquet/DruidParquetInputFormat.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.data.input.parquet;
+
+import org.apache.parquet.hadoop.ParquetInputFormat;
+
+import io.druid.data.input.InputRow;
+
+/**
+ * The implementation refers a lot to parquet-hadoop `ExampleInputFormat`
+ */
+public class DruidParquetInputFormat extends ParquetInputFormat<InputRow>
+{
+	public DruidParquetInputFormat(){
```

Comment:
this file is using tabs, please use 2-space indentation. pls see formatting related guidelines at https://github.com/druid-io/druid/blob/master/CONTRIBUTING.md